### DISCUSSION TITLE
When will the new version be released?

### DISCUSSION THREAD

**Discussion by baibai92** [d2phap/ImageGlass#920] [General] [answered]
Looking forward to the new release, can you tell me when it will be released?

**Comment by d2phap** [comment, marked as answer]
Hi @baibai92 

Probably on Dec 8, 2020. I haven't finished the preparation yet though.
In the meanwhile, the introduction video of v8.0 premieres today, at https://www.youtube.com/watch?v=GujLgI4bpuU

**Reply by baibai92** [reply]
Thanks, I'll go watch the video.
